lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -96,6 +96,12 @@ uwsgi:
   # Ensure application threads will run if `threads` is unset.
   enable-threads: true
 
+  # On some systems uWSGI has a default umask of 000, for Galaxy a 
```

Comment:
I think this also needs to be appended to https://github.com/mvdbeek/galaxy/blob/153ba0ce5151aaf120af9367316798ab4896bf46/lib/galaxy/config/config_manage.py#L165

Reply:
Sure.

Reply:
@bernt-matthias You really need to copy the whole config option documentation to the value of the `'desc'` key in `lib/galaxy/config/config_manage.py`, then execute `make config-rebuild` to update the sample files.

Reply:
OK, but if I do this then there are also changes in other docs. Should I just commit those as well?

Reply:
Yes, that'd be good.